docs/faq/frequently_asked_questions.rst

```diff
@@ -0,0 +1,56 @@
+
+.. include:: ../text_colors.rst
+.. toctree::
+
+.. _frequently_asked_questions:
+
+##########################
+Frequently Asked Questions
+##########################
+
+#. I just purchased my motor, how do I get started with Control Center? How can I communicate with my module?
+    * Your module must be configured properly through the Control Center before your module can spin. First, refer to your module’s family page that is found in the sidebar under the Modules section for hardware setup instructions. After your hardware is set up, refer to our :ref:`Control Center Manual <control_center_start_guide>` for instructions on how to set up the Control Center. Then, follow the instructions on configuring your module using Control Center depending on its firmware: :ref:`Speed <speed_module_start_guide>`, :ref:`Servo <servo_module_start_guide>`, or :ref:`Pulsing <pulsing_module_start_guide>`. Please see our documentation under the Tutorials section of the sidebar for additional information. We also have a supplementary application called :ref:`Vertiq Testing Tool <vertiq_testing_tool_guide>` to help you test your module.
```

Comment:
refer to your module's family page found under the sidebar's Modules section. Here, you will find the hardware setup instructions for your specific module.